package.json

```diff
@@ -10,10 +10,10 @@
     "cleanup": "rimraf dist",
     "lint": "eslint ./src --ext .ts",
     "migrate": "node -r dotenv/config dist/scripts/migrate.js",
-    "migrate:dev": "ts-node -r dotenv/config src/scripts/migrate.ts",
+    "migrate:dev": "ts-node -r dotenv/config src/scripts/migrate.ts | pino-pretty --translateTime 'UTC:yyyy-mm-dd\"T\"HH:MM:ss.l\"Z\"'",
```

Comment:
It looks like pino-pretty supports the idea of a config file at `pino-prettyrc` -- I think that would be nicer than sending config commands in here (both because of DRY but also this is now a much more verbose command than it was before!)

Reply:
Fixed up in [06139f2](https://github.com/PhilanthropyDataCommons/service/pull/45/commits/06139f2eebaa527dc1a163db055628c2936f3206)